zanata-war/src/main/java/org/zanata/webtrans/server/rpc/ActivateWorkspaceHandler.java

```diff
@@ -132,7 +132,7 @@ public ActivateWorkspaceResult execute(ActivateWorkspaceAction action,
         boolean isProjectObsolete = isProjectIterationObsolete(project.getStatus(), projectIteration.getStatus());
         boolean hasWriteAccess = hasWritePermission(project, locale);
         boolean hasGlossaryUpdateAccess = hasGlossaryUpdatePermission();
-        boolean requireReview = projectIteration.getRequireTranslationReview();
+        boolean requireReview = true;
```

Comment:
@huangp we are just enabling the review feature in editor regardless the boolean set in project version.  But I will check if this flag is used other than feature in editor.